meson_options.txt

```
@@ -387,3 +387,8 @@ option('efl-one',
   value : false,
   description : 'Build special set of .so´s as a single .so'
 )
+
+option('cmake_args',
+  type: 'string',
+  value : '',
+  description : 'Extra arguments for cmake when searching for dependencies.')
```

I think it is a good idea to stay in the pattern:
```suggestion

option('cmake_args',
  type: 'string',
  value : '',
  description : 'Extra arguments for cmake when searching for dependencies.'
)
```

---

Done.